content/07.pathogenesis.md

```diff
@@ -82,6 +82,10 @@ The study concludes that while large changes in R~t~ are observed, it is too ear
 
 More generally, population-level epidemic dynamics can be both observed and modelled.
 Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs or the comparison of predicted outcomes among prevention and control strategies.
+Many current efforts to model R~t~ have led to tools that assist the visualization of estimates in real time (or over recent intervals) [@url:https://epiforecasts.io/covid/; @url:https://rt.live/].
+While these may be valuable resources, it is important to note that they should be met with caution: the estimates arise from models containing many assumptions; and rely on the data quality, which varies widely by region.
```

Comment:
Looks good. Maybe just soften the note about caution somehow? Some of the preprints that we cite in the paragraphs above seem to be using very similar methods as the two "real-time" websites but they are mentioned without comment.

We could remove the caution part and just state the potential limitations:
```suggestion
While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions; and rely on the data quality, which varies widely by region.
```   

We could keep the caution part but mention why we are adding it here but not to papers that used similar methods:
```suggestion
While these may be valuable resources, it is important to note that they should be met with caution: the estimates arise from models containing many assumptions; and rely on the data quality, which varies widely by region; although these factors can be an issue for any model, real time visualization may not have the same rigorous evaluation of these factors that retrospective academic research may have.
```

Reply:
Also, should we put these two sentences at the end of the previous paragraph?

Reply:
thanks for your suggestions here - I take your point: why pick on 'live' estimates and not papers/preprints? I have no reason to think that the live models are of lesser quality than preprint estimates, but I think the public is far more likely to look at these live models -- with their nice visualization -- and I worry about public policy being informed by poorly understood modeling results. Thus my intent to express caution, though I do not want to veer into politics! Any help to achieve this end is appreciated.

Reply:
I think both your suggestions here are good; will wait for a little bit in case anyone else wants to chime in, before accepting them.